src/inspector/value-mirror.cc

```diff
@@ -1313,41 +1315,57 @@ std::unique_ptr<ValueMirror> createNativeSetter(v8::Local<v8::Context> context,
   return ValueMirror::create(context, function);
 }
 
+static const char* allowed_getters[] = {
+  "type", "fromElement", "target", "isTrusted"
+};
+
 bool doesAttributeHaveObservableSideEffectOnGet(v8::Local<v8::Context> context,
                                                 v8::Local<v8::Object> object,
                                                 v8::Local<v8::Name> name) {
-  // TODO(dgozman): we should remove this, annotate more embedder properties as
```

Comment:
The changes in this function don't seem related to the rest of this patch, and should be reverted.

Reply:
I don't think RUN-1475 will automatically fix these crashes but it will at least give us the mechanism needed to do so.  So in that case keeping the allowed getters for now is fine, but can we at least avoid commenting out tons of upstream code and just limit the logic changes we make to a new `if HasDivergedFromRecording` block at the beginning?

Reply:
Whoops, that `return` should have been in the `if`. My (very big) mistake! Thanks for catching that.

Reply:
@Domiii my concern is not with code being commented out, it's with us making large changes to the semantics of upstream V8.  This can cause unpredictable effects especially when we merge from upstream.  The only early return we should have is when we've diverged from the recording and hit the special logic you're adding for allowed getters.